src/FastRoute/src/DBWrapper.cpp

```diff
@@ -194,8 +194,7 @@ void DBWrapper::initRoutingTracks(std::vector<RoutingTracks>& allRoutingTracks,
     error("obd::dbTech not initialized\n");
   }
 
-  odb::dbBlock* block = _chip->getBlock();
-  if (!block) {
+  if (!_block) {
```

Comment:
boolean versus pointer again.
none of these checks should be this far down in the code. they should be in the tcl function
and assumed by the time control gets this far.  see OpenRoad.i db_has_tech() swig function for example